products/BellHybrid/apps/application-bell-settings/include/application-bell-settings/models/TimeUnitsModel.hpp

```diff
@@ -63,7 +63,7 @@ namespace app::bell_settings
         auto loadData() -> void override;
 
       private:
-        static constexpr int factoryResetTime    = 60 * 60 * 12; // noon
+        static constexpr int factoryResetTime    = 12; // noon
```

Comment:
This is then converted to time_t. Shouldn't it be just 0?

Reply:
Yep, it can be 0 as well :)